### PR TITLE
Developing the introduction

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**
Here is a barebones structure I wanted to give to the introduction page. I will keep adding to this PR as suggestions come in.

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -11,5 +11,121 @@ keypoints:
 ---
 The example-based nature of [Matplotlib documentation](https://matplotlib.org/) is GREAT.
 
-{% include links.md %}
+Matplotlib is the standard when it comes to making plots in Python. It is versatile and allows for lots of functionality and different ways to produce many plots.
+We will be focusing on using matplotlib for High Energy Physics.
+
+# A simple example
+A with any python code it is always a good practice to import the necessary libraries as a first step.
```
Comment (michmx) [inline _episodes/01-introduction.md]: Python should be written with capital P (that is what the official docs say).

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -11,5 +11,121 @@ keypoints:
...
+
+```python
+import matplotlib.pyplot as plt
+```
+
+Matplotlib graphs your data on Figures (i.e., windows, Jupyter widgets, etc.), each of which can contain one or more Axes (i.e., an area where points can be specified in terms of x-y coordinates, or theta-r in a polar plot, or x-y-z in a 3D plot, etc.). The simplest way of creating a figure with an axes is using pyplot.subplots. We can then use Axes.plot to draw some data on the axes:
```
Comment (michmx) [inline _episodes/01-introduction.md]: Is it better to say
"a figure with an axes" --> "a figure with axes" ?
Unless you mean an `Axes` object, which I would then write as the object.

**Review by michmx — CHANGES_REQUESTED** [review]
Looks good, I just include minor comments.

File: _episodes/01-introduction.md (hunk 1 above)
Comment (GuillermoFidalgo) [reply _episodes/01-introduction.md]: Agreed. Can your changes be automatically incorporated via Github or do I have to manually add them in a new commit?

Comment (michmx) [reply _episodes/01-introduction.md]: I am not very familiar with Github (we use Bitbucket at Belle II), but I think there is a way to just change directly on the page. In any case, a change directly in the page will create a commit, then it is the same.

File: _episodes/01-introduction.md (hunk 2 above)
Comment (GuillermoFidalgo) [reply _episodes/01-introduction.md]: Actually this is copy pasted from https://matplotlib.org/stable/tutorials/introductory/usage.html#a-simple-example. 
So I would blame the Matplotlib documentation 😆 but I get what you mean.

I would assume they meant to write 
> The simplest way of creating a figure with an Axes *object* is using `pyplot.subplots`. We can then use `Axes.plot` to draw some data on the axes:

**Review by GuillermoFidalgo — COMMENTED** [review]
Reviewed changes

**Review by michmx — APPROVED** [review]

**Pull request merged**